Makefile.Downstream.mk

```diff
@@ -1,3 +1,9 @@
+# defining variables here before including "Makefile" makes these variables unique for current makefile
+IMAGE_REGISTRY ?= quay.io
+REGISTRY_NAMESPACE ?= ocs-dev
+IMAGE_TAG ?= latest
+IMAGE_NAME ?= cephcsi-operator
+
```

Comment:
@iamniting PTAL

Reply:
We need to keep in mind to make changes here if we make any changes in the actual makefile.